docs/content/guides/developer/app-examples/trustless-swap.mdx

```diff
@@ -996,48 +1068,203 @@ The UI design consists of three parts:
+
+To streamline the styling process and keep the codebase clean and maintainable, we’ll be using Tailwind CSS, which provides utility-first CSS classes to rapidly build custom designs. Run the following command to add Tailwind CSS and its dependencies:
+
+<Tabs groupId="packagemanager">
+  <TabItem label="PNPM" value="pnpm">
+    ```bash
+    pnpm add tailwindcss@latest postcss@latest autoprefixer@latest
+    ```
+  </TabItem>
+  <TabItem label="Yarn" value="yarn">
+    ```bash
+    yarn add tailwindcss@latest postcss@latest autoprefixer@latest
+    ```
+  </TabItem>
+</Tabs>
+
+Next, generate the Tailwind CSS configuration file by running:
+
+```bash
+npx tailwindcss init -p
+```
 
-ReactDOM.createRoot(document.getElementById('root')!).render(
-	<React.StrictMode>
-		<Theme appearance="light">
-			<QueryClientProvider client={queryClient}>
-				<SuiClientProvider networks={networkConfig} defaultNetwork="testnet">
-					<WalletProvider autoConnect>
-						<RouterProvider router={router} />
-					</WalletProvider>
-				</SuiClientProvider>
-			</QueryClientProvider>
-		</Theme>
-	</React.StrictMode>,
-);
+<details>
+  <summary>
+    Replace the content of `tailwind.config.js` with the following:
+  </summary>
+  {@inject: examples/trading/frontend/tailwind.config.js}
+</details>
+
+<details>
+  <summary>
+    Add the `src/styles/` directory and add `base.css`: 
+  </summary>
+  {@inject: examples/trading/frontend/src/styles/base.css}
+</details>
+
+### Connecting your deployed package
+
+First, deploy your package via the [scripts in the api directory](#deployment).
+
+<details>
+  <summary>
+    Then, create a `src/constants.ts` file and fill it with the following:
+  </summary>
+  {@inject: examples/trading/frontend/src/constants.ts}
+</details>
+
+### Add helper functions and UI components
+
+<details>
+  <summary>
+    Create a `src/utils/` directory and add the following file: 
+  </summary>
+  {@inject: examples/trading/frontend/src/utils/helpers.ts}
+</details>
+
+Create a `src/components/` directory and add the following components: 
+
+<details>
+  <summary>
+    `ExplorerLink.tsx`
+  </summary>
+  {@inject: examples/trading/frontend/src/components/ExplorerLink.tsx}
+</details>
+
+<details>
+  <summary>
+    `InfiniteScrollArea.tsx`
+  </summary>
+  {@inject: examples/trading/frontend/src/components/InfiniteScrollArea.tsx}
+</details>
+
+<details>
+  <summary>
+    `Loading.tsx`
+  </summary>
+  {@inject: examples/trading/frontend/src/components/Loading.tsx}
+</details>
+
+<details>
+  <summary>
+    `SuiObjectDisplay.tsx`
+  </summary>
+  {@inject: examples/trading/frontend/src/components/SuiObjectDisplay.tsx}
+</details>
+
+### Set up routing
+
+The imported template only has a single page. To add more pages, you need to set up routing.
+
+First, install the necessary dependencies:
+
+<Tabs groupId="packagemanager">
+  <TabItem label="PNPM" value="pnpm">
+    ```bash
+    pnpm add react-router-dom react-hot-toast
```

Comment:
nit: Do we need to include `react-hot-toast` as a dependency at this stage? It seems like we could wait to add it (and the `<Toaster />` component for the first time we need to display a toast.

Reply:
I moved the `react-hot-toast` to the first instance of it being used which is in the previous step. The ExplorerLink uses it.